### ISSUE TITLE
Crashes on Windows in v1.9.0 and v1.9.1

### ISSUE THREAD

**Issue by Tienisto** [localsend/localsend#476]
According to #447 and discussion on Discord, some Windows users noticed that LocalSend crashes right at the start of the app.

Key observations:
- works in 1.8.0 but not in 1.9.0/1.9.1
- may work in 1.9.0 but accidentally stops working (after several days of use)
- deleting the app settings resolves the error
- when this error happens, the app does not show up and stays invisible
- settings file is corrupted with "0"s according to #487

**Comment by TheGB0077** [comment]
@SelaseKay Can you get this info: 
```json
"flutter.ls_window_offset_x": ,
"flutter.ls_window_offset_y": ,
"flutter.ls_window_height": ,
"flutter.ls_window_width":
```

out of your json file at `%AppData%\org.localsend\localsend_app`?

**Comment by SelaseKay** [comment]
Ok. I will try that out.

**Comment by TheGB0077** [comment]
@SelaseKay Paste it in here when you do, please!

**Comment by Tienisto** [comment]
According to #487, the file is actually corrupted. I have made the perstistence of window dimensions sequencial, lets see if it fixes the problem

**Comment by Tienisto** [comment]
This should be fixed in `v1.10.0`
